concourse/pipelines/create-deployer.yml

```diff
@@ -79,7 +79,8 @@ jobs:
     - get: paas-cf
     - task: try-fetch-bucket-state
       config:
-        image: docker:///governmentpaas/awscli
+        platform: linux
+        image: docker:///ubuntu
```

Comment:
There was a comment on an individual commit that hasn't been addressed yet. Making a note here, so we don't forget it: https://github.com/alphagov/paas-cf/commit/c30bb834dd01efeea29b47aa165e57dcac4aa499#diff-5eac5d6fc041d1da10144e10c0fa2ed0L82